Dockerfile

```diff
@@ -22,19 +22,13 @@ RUN go mod tidy
 RUN make install
 
 # Get Hermes build
-# FROM informalsystems/hermes:1.0.0 AS hermes-builder
-# remove line below and use the line above once hermes is updated
-FROM majita/hermes-unofficial:latest AS hermes-builder
+FROM informalsystems/hermes:1.2.0 AS hermes-builder
 
 FROM --platform=linux/amd64 fedora:36
 RUN dnf update -y
-RUN dnf install -y which iproute iputils procps-ng vim-minimal tmux net-tools htop jq openssl1.1
+RUN dnf install -y which iproute iputils procps-ng vim-minimal tmux net-tools htop jq
 USER root
 
-# Copy Hermes and IS binaries to final image
-COPY --chown=0:0 --from=hermes-builder /usr/lib/x86_64-linux-gnu/libssl.so.1.1 /usr/lib/x86_64-linux-gnu/libssl.so.1.1
```

Comment:
This dependency is no longer required. Hermes seems to have either bundled libssl with the binary or no longer depends on this specific version